code/modules/paperwork/clipboard.dm

```diff
@@ -1,154 +1,155 @@
+#define PAPERWORK	1
+#define PHOTO		2
+
 /obj/item/weapon/clipboard
 	name = "clipboard"
+	desc = "It looks like you're writing a letter. Want some help?"
 	icon = 'icons/obj/bureaucracy.dmi'
 	icon_state = "clipboard"
 	item_state = "clipboard"
-	throwforce = 0
 	w_class = WEIGHT_CLASS_SMALL
-	throw_speed = 3
-	throw_range = 10
-	var/obj/item/weapon/pen/haspen		//The stored pen.
-	var/obj/item/weapon/toppaper	//The topmost piece of paper.
+	var/obj/item/weapon/pen/containedpen
+	var/obj/item/weapon/toppaper
 	slot_flags = SLOT_BELT
 	burn_state = FLAMMABLE
 
 /obj/item/weapon/clipboard/New()
+	..()
 	update_icon()
 
-/obj/item/weapon/clipboard/MouseDrop(obj/over_object as obj) //Quick clipboard fix. -Agouri
-	if(ishuman(usr))
-		var/mob/M = usr
-		if(!(istype(over_object, /obj/screen) ))
-			return ..()
-
-		if(!M.restrained() && !M.stat)
-			switch(over_object.name)
-				if("r_hand")
-					M.unEquip(src)
-					M.put_in_r_hand(src)
-				if("l_hand")
-					M.unEquip(src)
-					M.put_in_l_hand(src)
-
-			add_fingerprint(usr)
+/obj/item/weapon/clipboard/verb/removePen()
+	set category = "Object"
+	set name = "Remove clipboard pen"
+	if(!ishuman(usr) || usr.incapacitated())
+		return
+	penPlacement(FALSE, containedpen)
+
+/obj/item/weapon/clipboard/proc/isPaperwork(var/obj/item/weapon/W) //This could probably do with being somewhere else but for now it's fine here.
+	if(istype(W, /obj/item/weapon/paper) || istype(W, /obj/item/weapon/paper_bundle))
+		return PAPERWORK
+	if(istype(W, /obj/item/weapon/photo))
+		return PHOTO
+
+/obj/item/weapon/clipboard/proc/isPen(var/obj/item/weapon/W) //Should also probably be somewhere else
```

Comment:
@Citinited if you're doing this just for readability, at the start of this file add:
```#define IS_PEN(x) (istype(x, /obj/item/weapon/pen))``` (it's standard to write macros like this all uppercase)
And at the end of the file, ```#undef IS_PEN```

The way you've done it at the moment, with a proc, is adding overhead where it's not necessary.